gateway-service/src/main/java/org/zowe/apiml/gateway/security/mapping/ExternalMapper.java

```diff
@@ -73,18 +72,25 @@ MapperResponse callExternalMapper(@NotNull HttpEntity payload) {
             if (httpResponse.getEntity() != null) {
                 response = EntityUtils.toString(httpResponse.getEntity(), StandardCharsets.UTF_8);
             }
-            if (statusCode < HttpStatus.SC_OK || statusCode >= HttpStatus.SC_MULTIPLE_CHOICES) {
-                log.warn("Unexpected response from the external identity mapper. Status: {} body: {}", statusCode, response);
+            if (statusCode == 0) {
+                return null;
+            }
+            if (!org.springframework.http.HttpStatus.valueOf(statusCode).is2xxSuccessful()) {
+                if (org.springframework.http.HttpStatus.valueOf(statusCode).is5xxServerError()) {
+                    log.error("Unexpected response from the external identity mapper. Status: {} body: {}", statusCode, response);
+                } else {
+                    log.debug("Unexpected response from the external identity mapper. Status: {} body: {}", statusCode, response);
```

Comment:
I guess there is no better way how to join it in one line, but the text could be stored in a class property...

Reply:
I will fix it